UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Helpers/CommonHelper.cs

```diff
@@ -13,5 +16,33 @@ public static string GetBase64EncodedCredentials(string userName, string passwor
         public static string ExtractAccessToken(string response) => response.Split(",")[0].Split(":")[1].Remove(0, 1).Replace("\"", "");
 
         public static string ExtractBatchId(string url) => new UriBuilder(url).Uri.Segments.FirstOrDefault(d => Guid.TryParse(d.Replace("/", ""), out Guid _));
+
+        public static string GetBlockIds(int blockNum)
+        {
+            string blockId = $"Block_{blockNum:00000}";
+            return blockId;
+        }
+
+        public static byte[] CalculateMD5(byte[] requestBytes)
+        {
+            using var md5 = MD5.Create();
+            var hash = md5.ComputeHash(requestBytes);
+
+            return hash;
+        }
+
+        public static byte[] CalculateMD5(Stream requestStream)
+        {
+            using var md5 = MD5.Create();
+            var hash = md5.ComputeHash(requestStream);
+
+            return hash;
+        }
+
+        public static int GetCurrentWeekNumber(DateTime date)
```

Comment:
Could this be simplified to an expression:

    => CultureInfo.InvariantCulture.Calendar.GetWeekOfYear(date, CalendarWeekRule.FirstFullWeek, DayOfWeek.Thursday);

Reply:
Simplified as per suggestion